test/integration.js

```diff
@@ -41,39 +40,38 @@ describe("Bing Web", function () {
                 {
                   top: 5,
                   market: 'en-US',
-                  adult: 'Strict',
-                  options: ['DisableLocationDetection', 'EnableHighlighting']
+                  adult: 'Strict'
                 },
                 function (err, res, body) {
 
       should.not.exist(err);
       should.exist(res);
       should.exist(body);
 
-      body.d.results.should.have.length(5);
+      body.webPages.value.should.have.length(5);
 
       done();
     });
   });
 
-  it('finds ukrainian results', function(done){
-    Bing.web('"Sony Xperia Z3" огляди тест',
+  it('finds russian results', function(done){
```

Comment:
ukranian is no longer supported :p I guess microsoft is capitalist, so they may assume Russia capitalized on Ukraine, kind of weird on them